http/headers/cross-origin-resource-policy.json

```diff
@@ -7,11 +7,17 @@
           "support": {
             "chrome": {
               "version_added": "73",
-              "notes": "Until version 75, downloads for files with this header would fail in Chrome. See <a href='https://crbug.com/952834'>bug 952834</a>."
+              "notes": [
+                "Until version 75, downloads for files with this header would fail in WebView. See <a href='https://crbug.com/952834'>bug 952834</a>.",
+                "From version 80, linearized PDFs served inline with this header fail to render properly. See <a href='https://crbug.com/1074261'>bug 1074261</a>."
+              ]
             },
             "chrome_android": {
               "version_added": "73",
-              "notes": "Until version 75, downloads for files with this header would fail in Chrome. See <a href='https://crbug.com/952834'>bug 952834</a>."
+              "notes": [
+                "Until version 75, downloads for files with this header would fail in WebView. See <a href='https://crbug.com/952834'>bug 952834</a>.",
```

Comment:
```suggestion
                "Until version 75, downloads for files with this header would fail in Chrome. See <a href='https://crbug.com/952834'>bug 952834</a>.",
```